HW_1/task_1.py

```diff
@@ -0,0 +1,23 @@
+"""
+Напишите программу, которая принимает на вход цифру, обозначающую день недели,
+и проверяет, является ли этот день выходным.
+
+Ввод: значение типа <int>
+Вывод: единственное значение типа <bool> (True либо False)
+
+Пример:
+6
+True
+
+7
+True
+
+1
+False
+"""
+
+number1 = int(input("Введите день недели: "))
+
+if number1 == 6 : print (True)
+elif number1 == 7 : print (True)
+else : print(False)
```

Comment:
Такой вариант записи кода возможен, однако согласно PEP8, он не рекомендуется.
```suggestion
if number1 == 6:
    print (True)
elif number1 == 7:
    print (True)
else:
    print(False)
```

Reply:
Два первых условия можно объединить.